modules/common/networking/hosts.nix

```diff
@@ -0,0 +1,74 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  config,
+  lib,
+  ...
+}: let
+  # please note that .100. network is not
+  # reachable from ghaf-host. It's only reachable
+  # guest-to-guest.
+  # Use to .101. (debug) to access guests from host.
+  # debug network hosts are post-fixed: <hostname>-debug
+  ipBase = "192.168.100";
+  debugBase = "192.168.101";
+  hostsEntries = [
+    {
+      ip = 1;
+      name = "net-vm";
+    }
+    {
+      ip = 2;
+      name = "ghaf-host";
+    }
+    {
+      ip = 3;
+      name = "gui-vm";
+    }
+    {
+      ip = 4;
+      name = "ids-vm";
+    }
+    {
+      ip = 5;
+      name = "audio-vm";
+    }
+    {
+      ip = 10;
+      name = "admin-vm";
+    }
+    {
+      ip = 100;
+      name = "chromium-vm";
+    }
+    {
+      ip = 101;
+      name = "gala-vm";
+    }
+    {
+      ip = 102;
+      name = "zathura-vm";
+    }
+    {
+      ip = 103;
+      name = "element-vm";
```

Comment:
In element-vm config element.nix:79 there is a refrence to a static ip address (of element-vm) 192.168.100.253 
The same ip is also set as internal config variable in netvmExtraModules.nix:33 (element-vmIp = "192.168.100.253";)
To me it seems that this is to get dendrite routing done with iptables on dendrite-pinecone.nix.
For now I think it should work if the ip is set correctly in all places but this (dendrite networking setup) would need a bit more love. :)

Reply:
This should be fixed now.